src/components/CalendarBodyForMonthView.tsx

```diff
@@ -121,8 +121,19 @@ function _CalendarBodyForMonthView<T>({
                 </Text>
                 {date &&
                   events
-                    .filter(({ start }) =>
-                      dayjs(start).isBetween(date.startOf('day'), date.endOf('day'), null, '[)'),
+                    .sort((a, b) => a.start.getUTCDate() - b.start.getUTCDate())
+                    .sort((a, b) => {
+                      const aDuration = dayjs.duration(dayjs(a.end).diff(dayjs(a.start))).days()
+                      const bDuration = dayjs.duration(dayjs(b.end).diff(dayjs(b.start))).days()
+                      return bDuration - aDuration
+                    })
```

Comment:
Can you do this at once? (one sort function)

Reply:
I will look into it!

Reply:
Thanks!!

Reply:
Done!